core/src/main/scala/sttp/openai/requests/audio/transcriptions/TranscriptionConfig.scala

```diff
@@ -0,0 +1,34 @@
+package sttp.openai.requests.audio.transcriptions
+
+import sttp.openai.requests.images.ResponseFormat
+
+import java.io.File
+import java.nio.file.Paths
+
+case class TranscriptionConfig(
+    file: File,
+    model: String,
```

Comment:
**suggestion:** How about making it an enum with `Whisper1` and `Custom` options, just like with the `Size` we discussed previously?

Reply:
Sure, sounds good, since for now it's the only available model to use